src/utils/utils.js

```diff
@@ -589,6 +600,16 @@ export function isInPre(node) {
             getComputedStyle(element).getPropertyValue('white-space') === 'pre')
     );
 }
+/**
+ * Returns true if the given node is or in a Node with ContentEditable false.
+ *
+ * @param {Node} node
+ * @returns {boolean}
+ */
+export function isContentUneditable(node) {
+    let element = node.nodeType === Node.TEXT_NODE ? node.parentElement : node;
```

Comment:
This confirms the need for an util to do just that, as mentioned above.

Reply:
I'm gonna refactor this and use it for the other review too